localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -251,9 +251,24 @@
         <trans-unit id="msgPromptFirewallRuleCreated">
             <source xml:lang="en">Firewall rule successfully created.</source>
         </trans-unit>
+        <trans-unit id="msgChooseQueryHistoryListing">
+            <source xml:lang="en">Choose Query History listing</source>
+        </trans-unit>
+        <trans-unit id="msgChooseQueryHistoryAction">
+            <source xml:lang="en">Choose An Action</source>
+        </trans-unit>
+        <trans-unit id="msgOpenQueryHistoryListing">
+            <source xml:lang="en">Open Query History Listing</source>
```

Comment:
I explicitly didn't want the names `Open Query` and `Run Query` because they clash with the normal commands to them. In order for the actions to have context, I think it's necessary to tell the user which query is being run or open. The context menu actions get their text from here - https://github.com/microsoft/vscode-mssql/pull/1561/files#diff-6e5249b55e8d29bc438070c22a9c9e34R11

What you are talking about is the text on command palette. Since this is the first pass so I can get this to the design review on Thursday, this is something that should come up on Thursday in the review with the PM's.

Reply:
That's fine - you can ignore the string comments until you have the design review then so that we have the full context of what the experience is going to look like.